website/content/docs/post-processors/vsphere.mdx

```diff
@@ -80,6 +80,39 @@ to a vSphere cluster.
 You can also use this post-processor with the vmx artifact from a vmware build.
 
 <Tabs>
+<Tab heading="HCL2">
```

Comment:
Neat ! https://packer-38ptvnalf-hashicorp.vercel.app/docs/post-processors/vsphere#example

Reply:
fixed the formatting on this